test/units/EverlongPortfolio.t.sol

```diff
@@ -31,7 +31,10 @@ contract TestEverlongPositions is EverlongTest {
     }
 
     function setUp() public virtual override {
+        TARGET_IDLE_LIQUIDITY_PERCENTAGE = 0.1e17;
+        MAX_IDLE_LIQUIDITY_PERCENTAGE = 0.2e17;
```

Comment:
If we're already using a non-standard exponent (17 instead of 18), why don't we just use `1e16` and `2e16`? IMO, it would be clearer to use `0.01e18` and `0.02e18` since that makes it immediately obvious that it's 1% and 2%.